core/bin/zksync_server/src/main.rs

```diff
@@ -38,8 +38,7 @@ struct Cli {
     /// Generate genesis block for the first contract deployment using temporary DB.
     #[arg(long)]
     genesis: bool,
-    /// Wait for the `setChainId` event during genesis.
-    /// If `--genesis` is not set, this flag is ignored.
+    /// Set chain id (temporary will be moved to genesis config)
```

Comment:
Let's leave "Wait for the `setChainId` event during genesis." comment. Also, why have you changed the behavior, i.e. "If `--genesis` is not set, this flag is ignored." is not true anymore, is there any reason?

Reply:
For now, it just doesn't work properly. 
I'll implement it properly once we have shared bridges